src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -312,6 +312,11 @@ internal void SendInvocationRequest(ScriptInvocationContext context)
                         if (pair.Value != null)
                         {
                             invocationRequest.TriggerMetadata.Add(pair.Key, pair.Value.ToRpc(_workerChannelLogger, _workerCapabilities));
+                            if (pair.Value is DefaultHttpRequest)
+                            {
+                                invocationRequest.TriggerMetadata[pair.Key].Http.Body = null;
```

Comment:
Add a new capability that does not sent DefaultHttpRequest as part of trigger metadata. Let's allow language workers to opt in into this change. Also, please open tracking issues with label: 3.x to remove this flag. Same applies to body/rawbody fix you did earlier. We we move to v3.x we can clean up proto buff and remove special flags.

Reply:
Opened issue for this https://github.com/Azure/azure-functions-host/issues/4813